src/generate-analysis.ts

```diff
@@ -35,7 +35,7 @@ interface Members {
   functions: Set<ResolvedFunction>;
 }
 
-export function generateElementMetadata(
+export function generateAnalysisMetadata(
```

Comment:
Man, `metadata` is not such a great word here. Too general.

It's kinda a serialization. Or maybe we could just call it an analysis?

Like, Documents and Packages are live and queryable, but an Analysis is a dead, serializable representation of the most interesting stuff?